extensions/amp-brightcove/0.1/amp-brightcove.js

```diff
@@ -34,7 +34,19 @@ class AmpBrightcove extends AMP.BaseElement {
   }
 
   /** @override */
+  buildCallback() {
+    /** @private {?Element} */
+    this.iframe_ = null;
+
+    /** @private {?Promise} */
+    this.iframePromise_ = null;
+  }
+
+  /** @override */
   layoutCallback() {
+    if (this.iframePromise_) {
```

Comment:
hmm actually I thought layout will be called multiple times. I think I got this impression from the tests having created multiple iframes - checking again it seems the test actually calls layoutCallback again manually without turning off runtime. Will fix that instead.